README.md

```diff
@@ -248,6 +248,21 @@ Hasura is an open source GraphQL engine that deploys instant, realtime GraphQL A
 - [Tutorial: Fullstack React Native with GraphQL and Authentication](https://blog.hasura.io/tutorial-fullstack-react-native-with-graphql-and-authentication-18183d13373a/)
 - [Guide: Hasura GraphQL Engine One-click App on DigitalOcean Marketplace](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html)
 
+## Developer Productivity [↗](https://awesomestacks.dev/developer-productivity)
```

Comment:
For the name, let's have one that mentions the tools and the use case, like "Monitoring Jenkins with Grafana"?

Reply:
Sound good but It doesn't reflect the complete use case.
CI-CD Jenkins-GitHub-Jira monitored by Grafana?

Reply:
Ah okay, I read the resources and I see what you mean. It looks like one use case is general Jenkins monitoring and another is the JIRA/GitHub workflow. The latter one seems the most primary so we could use that, how about his as the title: "Workflow automation with JIRA, GitHub and Jenkins"? It doesn't capture everything but the title space is limited. If you wanted, you could also break the Grafana/Prometheus stack out into another awesome stack, e.g. "Monitoring Jenkins with Grafana and Prometheus". The use case feels sufficiently different.

Reply:
Agree. In some hours I will address the request changes.